x-pack/solutions/security/plugins/security_solution/public/threat_intelligence/modules/indicators/components/common/field_value.tsx

```diff
@@ -41,7 +58,14 @@ export const IndicatorFieldValue: FC<IndicatorFieldValueProps> = ({ indicator, f
   return fieldType === 'date' ? (
```

Comment:
This return is also getting pretty difficult to read. I think we should divided a bit. You could do  an early return for the date case, and maybe another early return for the null value?